docs/kratos/self-service/flows/account-recovery-password-reset.mdx

```diff
@@ -0,0 +1,205 @@
+---
+id: account-recovery-password-reset
+title: Account recovery and password reset
```

Comment:
I feel like, here, it might be quite useful to have "Password reset" as well, as most people associate account recovery with password reset... But of course it's duplicated as well... Hm...